backend/Wheelingful.BLL/Models/Responses/FetchBookPaginationResponse.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Wheelingful.BLL.Models.Responses;
+
+public class FetchBookPaginationResponse
+{
+    public int PageCount { get; set; }
```

Comment:
include PageSize and TotalItems in response. If user set PageSizze in query with a huge value, your db will suffer. So you Limit the PageSize and return the updated value

Reply:
PageSize is already limited in validation of the request. As for totalItems - I don't quite understand the purpose.